http-client/Network/HTTP/Client/Request.hs

```diff
@@ -303,6 +304,7 @@ defaultRequest = Request
         , requestManagerOverride = Nothing
         , shouldStripHeaderOnRedirect = const False
         , proxySecureMode = ProxySecureWithConnect
+        , redactHeaders = Set.fromList [ "Authorization" ]
```

Comment:
Really pedantic comment, but `Set.singleton "Authorization"` would be more idiomatic and likely a tiny bit more efficient.